snuba/admin/jwt.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any, Optional
+
+import structlog
+
+from snuba.admin.user import AdminUser
+
+logger = structlog.get_logger().bind(module=__name__)
+
+
+CERTS: Optional[Any] = None
+AUDIENCE: Optional[str] = None
+
+
+def _certs() -> Any:
+    """Returns a dictionary of current Google public key certificates for
+    validating Google-signed JWTs. Since these change rarely, the result
+    is cached on first request for faster subsequent responses.
+    """
+    import requests
```

Comment:
Any specific reason you are importing modules locally within different function?

Reply:
I was just copying the GCP example -- happy to move out that import, they probably do that in case you copy/paste into a much bigger and higher traffic application:

https://cloud.google.com/python/docs/getting-started/authenticate-users#create_the_source_code